backend/interface.go

```diff
@@ -0,0 +1,33 @@
+package backend
+
+import (
+	"errors"
+	"io"
+	"io/fs"
+	"os"
+)
+
+var (
+	ErrIncorrectOpenMode = errors.New("disk file or device not open for write")
+	ErrNotSuitable       = errors.New("backing file is not suitable")
+)
+
+type File interface {
+	fs.File
+	io.ReaderAt
+	io.Seeker
+	io.Closer
+}
+
+type WritableFile interface {
+	File
+	io.WriterAt
+}
+
+type Storage interface {
+	File
+	// OS-stecific file for ioctl calls via fd
```

Comment:
Typo: "OS-specific"

Reply:
Whoopsie.. and in two different places! Copy-paste wins..

Fixed that, thank you!